### PR TITLE
Alert query for multiple object IDs

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR enables multiple object name alert queries.

### CODE REVIEW

File: extensions/skyportal/skyportal/handlers/api/alert.py [hunk 1]
```diff
@@ -493,12 +517,209 @@ def post_alert(
...
+    include_all_fields=False,
+    object_ids=None,
+):
+    """Get alert from database (by object ID).
+    ra : float
+        Right Ascension
+    dec : float
+        Declination
+    radius : float
+        Radius to search
```
Comment (guynir42) [inline extensions/skyportal/skyportal/handlers/api/alert.py:650]: Specify the units (in the case of radius, say that units can be selected separately but specify what is the default).

File: extensions/skyportal/skyportal/handlers/api/alert.py [hunk 2]
```diff
@@ -493,12 +517,209 @@ def post_alert(
...
+    radius_units : str
+        Units of radius variable. Can be deg, arcmin, or arcsec.
+    program_id_selector : List[Int]
+        List of Program IDs to query
+    projection : List[str]
+        List of fields to query from database
+    include_all_fields : bool
+        Include all fields in database. Defaults to False.
```
Comment (guynir42) [inline extensions/skyportal/skyportal/handlers/api/alert.py:658]: I'm not sure what "all fields" means.

**Review by guynir42 — APPROVED** [review]
Minor comments on the docstrings but otherwise looks fine.

**Review by guynir42 — APPROVED** [review]
Minor comments on the docstrings but otherwise looks fine.

File: extensions/skyportal/skyportal/handlers/api/alert.py (hunk 1 above)
Comment (mcoughlin) [reply extensions/skyportal/skyportal/handlers/api/alert.py]: Good point.

File: extensions/skyportal/skyportal/handlers/api/alert.py (hunk 2 above)
Comment (mcoughlin) [reply extensions/skyportal/skyportal/handlers/api/alert.py]: Yes.

**Pull request merged**